### ISSUE TITLE
Make the whole component accessible:  (WCAG) international standard

### ISSUE THREAD

**Issue by justoverclockl** [justoverclockl/react-fast-scheduler#13]
The whole library should be accessible, so we neet to implement the  (WCAG) international standard.
Also check markup and use semantic elements correctly
